File: assets/samples/reverse.whois/key.value/php/reverse.key.value.post.php
<?php

$user = 'your whois api username';
$password = 'your whois api password';

$header = "Content-Type: application/json\r\nAccept: application/json\r\n";

$options = array('http' => array(
    'method' => 'POST',
    'header' => $header
));

$url = 'https://www.whoisxmlapi.com/reverse-whois-api/search.php';

$options['http']['content'] = json_encode(array(
    'terms'=>array(
        array(
            'section' => 'Registrant',
            'attribute' => 'Email',
            'value' => 'a@b.com',
            'exclude' => false
        )
    ),
    'recordsCounter' => false,
    'username' => $user,
    'password' => $password
));

print(file_get_contents($url, false, stream_context_create($options)));

File: assets/samples/reverse.whois/key.value/python/reverse.key.value.post.py
try: 
    import http.client as http
except:
    import httplib as http

password = 'your whois api password'
username = 'your whois api username'

payload = '{ "terms": [{ "section": "Registrant", "attribute": "Email",\
                         "value": "a@b.com", "exclude": false }],\
             "recordsCounter": false, "username": "USER", "password": "PASS"\
           }'.replace('USER', username).replace('PASS', password)

headers = {"Content-Type": "application/json", "Accept": "application/json"}
conn = http.HTTPSConnection('www.whoisxmlapi.com')
conn.request("POST", '/reverse-whois-api/search.php', payload, headers)

response = conn.getresponse()
text = response.read()
print(text)